doc/modules/cassandra/pages/managing/operating/metrics.adoc

```diff
@@ -528,17 +528,17 @@ ViewReplicasSuccess.
 base table and when CL.ONE is achieved on view.
 |===
 
-== Cache Metrics
+== Weighted Cache Metrics
```

Comment:
Should we also add a few words about 'weighted' and 'unweighted' terms? I think here is the comment with a well-described difference, we can use it:

https://issues.apache.org/jira/browse/CASSANDRA-17062?focusedCommentId=17443143&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17443143

Reply:
done